test/CMakeLists.txt

```diff
@@ -252,7 +252,7 @@ foreach(SDK ${SWIFT_SDKS})
     set(test_dependencies)
     get_test_dependencies("${SDK}" test_dependencies)
 
-    if(SWIFT_BUILD_STDLIB AND SWIFT_INCLUDE_TESTS)
+    if(SWIFT_BUILD_DYNAMIC_STDLIB AND SWIFT_INCLUDE_TESTS)
```

Comment:
Can you add a comment here explaining for future readers what is happening? Explaining why this is wanted for dynamic stdlib and the same inverse question for static stdlib.

Reply:
Added comment.